websubhub-native/src/main/java/io/ballerina/stdlib/websubhub/RequestHandler.java

```diff
@@ -0,0 +1,12 @@
+package io.ballerina.stdlib.websubhub;
+
+import io.ballerina.runtime.api.values.BObject;
+
+/**
+ * {@code RequestHandler} is a wrapper object used for service method execution.
+ */
+public class RequestHandler {
```

Comment:
Any reason for this ?

Reply:
* We could not initialize an object which is not isolated in an isolated-object. In this case `websubhub:Service` was not isolated so we need to have a wrapper around it so we could just invoke remote methods in the `websubhub:Service` class using that.

Reply:
* Moving all the native method calls to `RequestHandler` as per the discussion.